src/redux/slices/userSlice.ts

```diff
@@ -0,0 +1,48 @@
+import { createSlice, PayloadAction } from "@reduxjs/toolkit";
+import { Graph, Profile } from "../../utilities/types";
+
+interface userState {
+  profile: Profile | null;
+  graph: Graph | null;
+}
+
+const initialState: userState = {
+  profile: null,
+  graph: null,
+};
+
+export const userSlice = createSlice({
```

Comment:
I don't see anything in here that is storing the session into local/session storage. Is that happening automagically?

Reply:
I was going to do that in a separate story.

Reply:
https://www.pivotaltracker.com/story/show/177602937